share/completions/magento.fish

```diff
@@ -0,0 +1,720 @@
+	echo hr_HR\t"Croatian (Croatia)"
+	echo cs_CZ\t"Czech (Czech Republic)"
+	echo da_DK\t"Danish (Denmark)"
+	echo nl_BE\t"Dutch (Belgium)"
+	echo nl_NL\t"Dutch (Netherlands)"
+	echo en_AU\t"English (Australia)"
+	echo en_CA\t"English (Canada)"
+	echo en_IE\t"English (Ireland)"
+	echo en_NZ\t"English (New Zealand)"
+	echo en_GB\t"English (United Kingdom)"
+	echo en_US\t"English (United States)"
+	echo et_EE\t"Estonian (Estonia)"
+	echo fil_PH\t"Filipino (Philippines)"
+	echo fi_FI\t"Finnish (Finland)"
+	echo fr_BE\t"French (Belgium)"
+	echo fr_CA\t"French (Canada)"
+	echo fr_FR\t"French (France)"
+	echo gl_ES\t"Galician (Spain)"
+	echo ka_GE\t"Georgian (Georgia)"
+	echo de_AT\t"German (Austria)"
+	echo de_DE\t"German (Germany)"
+	echo de_CH\t"German (Switzerland)"
+	echo el_GR\t"Greek (Greece)"
+	echo gu_IN\t"Gujarati (India)"
+	echo he_IL\t"Hebrew (Israel)"
+	echo hi_IN\t"Hindi (India)"
+	echo hu_HU\t"Hungarian (Hungary)"
+	echo is_IS\t"Icelandic (Iceland)"
+	echo id_ID\t"Indonesian (Indonesia)"
+	echo it_IT\t"Italian (Italy)"
+	echo it_CH\t"Italian (Switzerland)"
+	echo ja_JP\t"Japanese (Japan)"
+	echo km_KH\t"Khmer (Cambodia)"
+	echo ko_KR\t"Korean (South Korea)"
+	echo lo_LA\t"Lao (Laos)"
+	echo lv_LV\t"Latvian (Latvia)"
+	echo lt_LT\t"Lithuanian (Lithuania)"
+	echo mk_MK\t"Macedonian (Macedonia)"
+	echo ms_Latn_MY\t"Malay (Malaysia)"
+	echo mn_Cyrl_MN\t"Mongolian (Mongolia)"
+	echo nb_NO\t"Norwegian Bokmål (Norway)"
+	echo nn_NO\t"Norwegian Nynorsk (Norway)"
+	echo fa_IR\t"Persian (Iran)"
+	echo pl_PL\t"Polish (Poland)"
+	echo pt_BR\t"Portuguese (Brazil)"
+	echo pt_PT\t"Portuguese (Portugal)"
+	echo ro_RO\t"Romanian (Romania)"
+	echo ru_RU\t"Russian (Russia)"
+	echo sr_Cyrl_RS\t"Serbian (Serbia)"
+	echo sk_SK\t"Slovak (Slovakia)"
+	echo sl_SI\t"Slovenian (Slovenia)"
+	echo es_AR\t"Spanish (Argentina)"
+	echo es_CL\t"Spanish (Chile)"
+	echo es_CO\t"Spanish (Colombia)"
+	echo es_CR\t"Spanish (Costa Rica)"
+	echo es_MX\t"Spanish (Mexico)"
+	echo es_PA\t"Spanish (Panama)"
+	echo es_PE\t"Spanish (Peru)"
+	echo es_ES\t"Spanish (Spain)"
+	echo es_VE\t"Spanish (Venezuela)"
+	echo sw_KE\t"Swahili (Kenya)"
+	echo sv_SE\t"Swedish (Sweden)"
+	echo th_TH\t"Thai (Thailand)"
+	echo tr_TR\t"Turkish (Turkey)"
+	echo uk_UA\t"Ukrainian (Ukraine)"
+	echo vi_VN\t"Vietnamese (Vietnam)"
+	echo cy_GB\t"Welsh (United Kingdom)"
+end
+
+
+function __fish_print_magento_source_theme_file_types -d "Shows all available source theme file types"
+  echo less\t"Currently, LESS is the only file type supported"
+end
+
+function __fish_print_magento_deploy_modes -d "Shows all available deploy modes"
+  echo developer\t"Development mode"
+  echo production\t"Production mode"
+end
+
+###
+# Retrieves all existing magento admin users from the mysql database
+###
+function __fish_print_magento_users -d "Shows all existing magento admin users"
+  set -l users (__fish_print_magento_mysql_command 'select username from admin_user;')
+
+  for i in $users
```

Comment:
Like above, this can just be `printf '%s\n' $users` or no variable assignment/ command substitution at all.